hardhat.config.ts

```diff
@@ -0,0 +1,154 @@
+import "@nomiclabs/hardhat-waffle";
+import "@nomiclabs/hardhat-ethers";
+import "@tenderly/hardhat-tenderly";
+import "@nomiclabs/hardhat-etherscan";
+import "@nomiclabs/hardhat-web3";
+import "hardhat-deploy";
+import "hardhat-deploy-ethers";
+import "@typechain/hardhat";
+
+import { resolve } from "path";
+import { config as dotenvConfig } from "dotenv";
+import { HardhatUserConfig } from "hardhat/config";
+import { NetworkUserConfig } from "hardhat/types";
+import { utils } from "ethers";
+import Web3 from "web3";
+
+dotenvConfig({ path: resolve(__dirname, "./.env") });
+
+const chainIds = {
+  ganache: 1337,
+  goerli: 5,
+  hardhat: 31337,
+  kovan: 42,
+  mainnet: 1,
+  rinkeby: 4,
```

Comment:
@pradyuman-verma I think we can remove testnets, because we don't have DSA deployed on testnets

Reply:
okay, on it

Reply:
we are using it in `createTestnetConfig`, so don't need to remove it I guess

Reply:
Ahh! okay. Then I think we can remove `createTestnetConfig` function completely.

Reply:
Okay, but let it be there because I deploy contracts first on the test net and then on the mainnet.
It will be helpful to first-time contributors/beginners.